client/reader/search-stream/blank.jsx

```diff
@@ -6,38 +6,16 @@ import React from 'react';
 /**
  * Internal Dependencies
  */
-import { localize } from 'i18n-calypso';
-import Suggestion from './suggestion';
-
-export function BlankContent( { translate, suggestions } ) {
-	let suggest = null;
-	if ( suggestions ) {
-		let sugList = suggestions
-			.map( function( query ) {
-				return (
-					<Suggestion suggestion={ query } />
-				);
-			} );
-		sugList = sugList
-			.slice( 1 )
-			.reduce( function( xs, x ) {
-				return xs.concat( [ ', ', x ] );
-			}, [ sugList[ 0 ] ] );
-
-		suggest = (
-			<p className="search-stream__blank-suggestions">
-				{ translate( 'Suggestions: {{suggestions /}}.', { components: { suggestions: sugList } } ) }
-			</p> );
-	}
 
+// TODO post-refresh: delete this file
```

Comment:
You can add this to the post-Refresh list here:

https://github.com/Automattic/wp-calypso/issues/9364